apps/api/src/app/config/__tests__/utilities/buildMockConfigService.ts

```diff
@@ -9,6 +9,18 @@ export default (configOverrides: ConfigOverrides, envFilePath: string) => {
     const realConfig = dotenv.parse(fs.readFileSync(envFilePath));
 
     const mockedConfig = {
```

Comment:
Add a todo to make this extensible. Maybe we could use class transformer in conjunction with our data class that we use for validating the environment?

Reply:
done